docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -447,6 +448,12 @@ Set the label process type for the container processes.
 Use a Podman secret in the container either as a file or an environment variable.
 This is equivalent to the Podman `--secret` option and generally has the form `secret[,opt=opt ...]`
 
+### `ShmSize=`
+
+Size of _/dev/shm_. A _unit_ can be **b** (bytes), **k** (kibibytes), **m** (mebibytes), or **g** (gibibytes).
```

Comment:
We try not to copy/paste the documentation from that of `podman-run`. Instead, just say that it is the equivalent of `--shm-size`

Reply:
I don't like forcing users to open multiple man pages to figure out how a function will work.  I guess we need to work with @edsantiago scripts to figure a way to allow podman-systemd.unit to import the same text as podman run and podman create man pages. @edsantiago What do you think?

Reply:
I understand your point. But, unless we use a single source of truth, maintaining the documentation will become very hard

Reply:
Yup,

Reply:
But we are pushing the pain onto the users rather then having the developers handle the pain.

Reply:
I agree. But, I think at this stage it's better to align to what's already there (i.e. reference to the podman command) and then solve it as a whole

Reply:
Well, the first thing I see here is that none of this makes any sense whatsoever.

In the man page we have:
```
--shm-size=number[unit]
...blah blah a unit can be
```
Here we have
```
ShmSize=
blah blah a unit can be     <<<<-----WTF is a "unit"? Where did "unit" come from?
```
My general sense is that this man page does not (ATM) include useful context hints like the man pages. Therefore copy-pasting (or some new TBD include mechanism) is impractical, even anti-helpful. Until that problem is solved, I suggest sticking with "see other man page".